vsts/pipelines/nightly.yml

```diff
@@ -1,16 +1,65 @@
-queue:
-  name: OryxLinux
+resources:
+- repo: self
+  fetchDepth: 15
+jobs:
+- job: Job_1
```

Comment:
Can these jobs have a good name (display name  is ok, but when reading 'depends on' it is kind of not clear immediately)